workflows/diagnostics/fv3net/diagnostics/prognostic_run/compute.py

```diff
@@ -523,16 +538,22 @@ def main(args):
     )
 
     # maps
-    diags["pwat_run_initial"] = input_data["2d"][0].PWAT.isel(time=0)
-    diags["pwat_run_final"] = input_data["2d"][0].PWAT.isel(time=-2)
-    diags["pwat_verification_final"] = input_data["2d"][0].PWAT.isel(time=-2)
+    diags["pwat_run_initial"] = (
+        input_data["2d"][0].PWAT.isel(time=0).rename({"time": "initial_time"})
+    )
+    diags["pwat_run_final"] = (
+        input_data["2d"][0].PWAT.isel(time=-2).rename({"time": "final_time"})
+    )
+    diags["pwat_verification_final"] = (
+        input_data["2d"][0].PWAT.isel(time=-2).rename({"time": "final_time"})
```

Comment:
These names (`initial_time` and `final_time`) are not quite accurate. Maybe call the former `first_output_time` and select the `time=-1` index for the latter two so that they are accurate?